src/analysis/pass/filter-destroy-ghidra/filter-destroy-ghidra

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# filter-destroy-ghidra: find results that only ghidra gets right (or wrong)
+
+require "json"
+
+# TODO(ww): Remove this.
+XED_SO = "./src/worker/xed/xed.so"
+ZYDIS_SO = "./src/worker/zydis/zydis.so"
+GHIDRA_SO = "./src/worker/ghidra/ghidra.so"
```

Comment:
I copied this from the other `filter-destroy-*` scripts. Not sure if there is a preference or reason for choosing XED and Zydis as the other two to compare.

Reply:
The general preference is for decoders that are (subjectively) "high quality", which both XED and Zydis are. So this LGTM!